client/state/ui/editor/media/imageEditor/constants.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import keyMirror from 'key-mirror';
```

Comment:
[I'm with @blowery](https://github.com/Automattic/wp-calypso/pull/751#discussion_r46969902) on abolishing `key-mirror`, but it's a minor detail.